src/BUILD

```diff
@@ -540,6 +542,26 @@ cc_binary(
     linkstatic = True,
 )
 
+# POC which can start OVMS with C-API built by bazel
+# Standalone example for building outside of bazel (with bare g++ is inside MakefileCapi)
+cc_binary(
```

Comment:
Add temporary additional target to dockerfile to verify its working?

Reply:
done